config/megan/HEMCO_sa_Grid.rc

```diff
@@ -0,0 +1,8 @@
+# Emission grid specifications:
+XMIN: -160.0
+XMAX:  -30.0
+YMIN:  7
+YMAX:  82
+NX: 1250
+NY: 750
+NZ: 1
```

Comment:
missing newline char at end of file (that's what the red symbol is indicating)

Reply:
it works